apps/dashboard/src/app/account/billing/subscriptions/subscriptions.component.ts

```diff
@@ -7,7 +7,6 @@ import { AccountService, STFTrackingService } from '@flaps/core';
 import { COUNTRIES } from '../utils';
 import { Currency } from '../billing.models';
 import { WINDOW } from '@ng-web-apis/common';
-import { SubscribedAccountDeleteComponent } from '../../account-manage/account-delete/subscribed-account-delete.component';
 import { DeprecatedCalculatorComponent } from '../calculator/deprecated-calculator.component';
```

Comment:
Is this normal that we're still using a component marked as deprecated? `DeprecatedCalculatorComponent`

Reply:
This deprecated component is not used any more. It was used for the old subscription plans. There's a pending task to remove it (sc-6568)